<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>suggested_time</class>
 <widget class="QWidget" name="suggested_time">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Suggested Times</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(76, 76, 76);
font: 12pt &quot;Courier&quot;;</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Courier&quot;;
background-color: rgb(255, 0, 128);</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_time">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Courier&quot;;
background-color: rgb(61, 229, 229);</string>
   </property>
   <property name="text">
    <string>Select Time</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_times">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>281</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 15pt &quot;Courier&quot;;</string>
   </property>
   <property name="text">
    <string>   Suggested Times:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
